block_element.go

```diff
@@ -3,26 +3,26 @@ package slack
 // https://api.slack.com/reference/messaging/block-elements
 
 const (
-	metImage      MessageElementType = "image"
-	metButton     MessageElementType = "button"
-	metOverflow   MessageElementType = "overflow"
-	metDatepicker MessageElementType = "datepicker"
-	metSelect     MessageElementType = "static_select"
-
-	mixedElementImage MixedElementType = "mixed_image"
-	mixedElementText  MixedElementType = "mixed_text"
+	MetImage      MessageElementType = "image"
```

Comment:
lets keep acronyms in caps.

Reply:
will update this for all occurrences but probably tomorrow since I'm about to sleep

Reply:
done ^